config/default.schema.json

```diff
@@ -334,6 +334,15 @@
 					"default": false,
 					"description": "send file to antivirus by uploading"
 				}
+			},
+			"default": {
+				"S3_ENDPOINT": "http://localhost:9000",
+				"S3_ACCESS_KEY_ID": "miniouser",
+				"S3_SECRET_ACCESS_KEY": "miniouser",
+				"S3_BUCKET": "schulcloud",
+				"S3_REGION": "eu-central-1",
+				"INCOMING_REQUEST_TIMEOUT": 600000,
+				"EXCHANGE": "files-storage"
 			}
```

Comment:
Why set you the defauts for S3?

Reply:
The problem is that I need the FILE_STORAGE__EXCHANGE variable to initialize the RabbitMqModule and the ToolsModule also needs FILES_STORAGE__SERVICE_BASE_URL, but the default value defined in the default.shema.json for it is not set, unless you set a default for the FILE_STORAGE-object, since the FILE_STORAGE-object requires that S3_ENDPOINT, S3_REGION, S3_BUCKET, S3_ACCESS_KEY_ID and S3_SECRET_ACCESS_KEY are set, otherwise the validation of FILE_STORAGE fails.

Reply:
mh okay for the deployed instances this defaults are wrong, are the real one needed?